docs/model/evidence/clinical_significance.rst

```diff
@@ -99,7 +99,7 @@ The available options for clinical significance depend on the evidence type sele
      - Definition
    * - Gain of Function
      - |arrow-circle-up|
-     - A sequence variant whereby new or enhanced function is conferred on the gene product
+     - A sequence variant whereby enhanced function is confered on the gene product
```

Comment:
`conferred` was correct.
https://www.merriam-webster.com/dictionary/confer
https://oed.com/view/Entry/38737

Reply:
I created another commit to revert spelling of conferred